domain/src/main/java/com/teamagam/gimelgimel/domain/alerts/UpdateLatestInformedAlertTimeInteractor.java

```diff
@@ -0,0 +1,39 @@
+package com.teamagam.gimelgimel.domain.alerts;
+
+import com.google.auto.factory.AutoFactory;
+import com.google.auto.factory.Provided;
+import com.teamagam.gimelgimel.domain.alerts.entity.Alert;
+import com.teamagam.gimelgimel.domain.alerts.repository.InformedAlertsRepository;
+import com.teamagam.gimelgimel.domain.base.executor.ThreadExecutor;
+import com.teamagam.gimelgimel.domain.base.interactors.BaseDataInteractor;
+import com.teamagam.gimelgimel.domain.base.interactors.DataSubscriptionRequest;
+
+import java.util.Collections;
+
+import rx.Observable;
+
+@AutoFactory
+public class UpdateLatestInformedAlertTimeInteractor extends BaseDataInteractor {
```

Comment:
Don't we need a simpler class for this functionality?
It's not real "subsription request", we just want to update the InformedAlertsRepository.

Maybe I just don't understand the real meaning of this kind of class. Isn't it  just for long streams like messages streams?